### ISSUE TITLE
Output is "Final exit status: 0" even if script failed

### ISSUE THREAD

**Issue by sypets** [t3docs/docker-render-documentation#92]
## Reproduce

1. Make Documenation-GENERATED-temp readonly (just to simulate an error):
`chmod 000 Documentation-GENERATED-temp/`

2. Run Docker and build with usual producedure ...

Output is: 

```
We saw these exitcodes (code, count):
{
  "0": 44,
  "1": 1,
  "22": 11
}

Final exit status: 0 (completed)

```

Also the exit code is 0 (which is also not correct)

`# echo $?`

Output:
`0`

**Comment by marble** [comment]
See also #22

**Comment by marble** [comment]
Solved.
Since Toolchain v2.10.0 (2020-05-08) and Container v2.6.1
we have:

* Use 0 or 255 for FINAL_EXIT_CODE, indication whether html build produced output

**Comment by marble** [comment]
Well, probably this isn't really solved yet. and very hard to do. 

The best thing to do to checker for a proper or sufficient build probably is: Check file `Documentation-GENERATED-temp/Result/project/0.0.0/_buildinfo/results.json` after the build has been done and decide yourself what should be counted as success.
 If `results.json` exists and has 

    {
      "created": {
        "html": 1
      },

this means that Sphinx successfully completed the HTML build. That does not garantee however that no warnings or errors were encountered.

**Comment by marble** [comment]
Correction. I now think it is actually solved with container v2.7. And there is explicite explanation in the output:
<pre>   ------------------------------------------------
   FINAL STATUS is: SUCCESS (exitcode 0)
                    because HTML builder succeeded
   ------------------------------------------------
   exitcode:   0            44 ms

==================================================
   90-Finish/run_20-Remove-lock.py
   exitcode:   0            55 ms

We saw these exitcodes (code, count):
{
  "0": 82
}
exiting with exitcode 0

Process finished with exit code 0
</pre>

So the FINAL_EXIT_CODE of the BUILD RUN of the container should either be zero (html was built) or 255 (html was not built).
